newIDE/app/src/EventsSheet/ParameterFields/GenericExpressionField/index.js

```diff
@@ -193,6 +193,10 @@ export default class ExpressionField extends React.Component<Props, State> {
     }
   }
 
+  componentWillUnmount() {
```

Comment:
a react warning was being shown because of the debounce. I assume closing the field faster than having the autocompletion returned triggered it

Reply:
Always a good idea to cancel a debounce (and to a point, anything async like a promise)